types/ckeditor__ckeditor5-engine/src/conversion/viewconsumable.d.ts

```diff
@@ -34,7 +35,7 @@ export default class ViewConsumable {
     ): void;
     revert(element: Text | DocumentFragment): void;
     test(
-        element: Element,
+        element: Text | Element | DocumentFragment | Node,
```

Comment:
`Element`, `Text`, and `DocumentFragment` all extend `Node`, so `Node` should be enough, right?

Reply:
`DocumentFragment` doesn't exist `Node`, but yes this could be simplified to `element: Node | DocumentFragment`...I'll make that change.

I made a note about your similar comment about `type Item`. Because that's used as the return value in a few places, it needs to be a union of Node and the classes that extend Node for the typescript checker to be able to correctly narrow the type.